RegionX-Labs/CoreHub
# Ismp integration

Testing steps:

1. Build the [RegionX-Node](https://github.com/RegionX-Labs/RegionX-Node)
2. Follow the [Getting started with zombienet](https://github.com/RegionX-Labs/Coretime-Mock?tab=readme-ov-file#getting-started-with-zombienet) steps and run `npm run zombienet`
3. Set up the environment by running the [initialization script](https://github.com/RegionX-Labs/Coretime-Mock?tab=readme-ov-file#example-setting-up-the-full-environment) after the parachains started producing blocks.
4. Set the environment variables to the following:
```.env
WS_KUSAMA_RELAY_CHAIN="ws://127.0.0.1:9900"
WS_KUSAMA_CORETIME_CHAIN="ws://127.0.0.1:9910"
WS_REGIONX_CHAIN="ws://127.0.0.1:9920"
EXPERIMENTAL=true
```
5. Run `npm run dev`
6. Purchase a core from the bulk market
7. Cross chain transfer the core from the Coretime chain to the RegionX parachain.
8. Upon successful transfer the region record should be set on the RegionX parachain, and you should see the region location set to `RegionX chain`.
